GraphLayout/tools/WpfGraphControl/GraphViewer.cs

```diff
@@ -746,7 +746,10 @@ void ProcessGraphUnderLock() {
 
                 CancelToken = new CancelToken();
 
-                if (_drawingGraph == null) return;
+                if (_drawingGraph == null) {
+                    ClearGraphViewer();
+                    return;
```

Comment:
I am just not sure whether `GraphChanged`/`LayoutComplete` events should still miss here. In case we do it, should we do also check for `RunLayoutAsync`?

Reply:
It could be safer on assigning a null to create an empty graph and process it instead.

Reply:
I updated it this way.